test/e2e/v3/plugin_cluster_test.go

```diff
@@ -166,7 +171,7 @@ func Run(kbc *utils.TestContext) {
 	_, err = kbc.Kubectl.Command(
 		"create", "clusterrolebinding", fmt.Sprintf("metrics-%s", kbc.TestSuffix),
 		fmt.Sprintf("--clusterrole=e2e-%s-metrics-reader", kbc.TestSuffix),
-		fmt.Sprintf("--serviceaccount=%s:default", kbc.Kubectl.Namespace))
+		fmt.Sprintf("--serviceaccount=%s:%s", kbc.Kubectl.Namespace, kbc.Kubectl.ServiceAccount))
```

Comment:
This is the kind of PR that will require changes in the SDK side such as: 

- In the go e2e tests
- Apply the same feature for ansible/helm
- Change the e2e test for ansible and helm. 

So, instead of manually change ansible and helm regards the changes in the config then, I'd like to see if we can first:

- merge 1.5 https://github.com/kubernetes-sigs/kubebuilder/pull/2060 / sdk alignment operator-framework/operator-sdk#4581.
- then push a pr in kb to centralize the code https://github.com/kubernetes-sigs/kubebuilder/issues/2015 and bump SDK with where Ansible/Helm/GO will use all the same code and let us know to close the issue: https://github.com/operator-framework/operator-sdk/issues/4542
- For them, we get this merged. Since it will mean change the e2e test for all types on the SDK side as well. 

See that how much changes like this one we merge before that, harder will be for us to align and keep maintained the SDK side. 

c/c @Adirio @jmrodri

Reply:
The addition of a service account is straightforward, improves default security, and is an often requested feature by users. The PR’s/steps you linked are unrelated and need a lot of review. Other PR’s aren’t being blocked by them so the argument that this one should doesn’t hold water.

Reply:
So, are you planning to get it merged and as follow up; 
- bump kb in sdk
- then, to do the config of  helm and ansible plugin
- then, to do changes in the tests for both + go/v2 and go/v3?

Reply:
Yep!

Reply:
Is that true though? Why would merging this now prevent ansible/helm plugins from receiving the same changes later through phase 1.5 or some other machination mentioned above? Wouldn't they just get this feature for free then?

Reply:
It would definetely work the way you mention. I just meant that once phase 1.5 is implemented, the process will be much easier as changing them in one place will update all (after a kubebuilder bump). I think the point of this conversation was just to show how right now they need to be updated in 3 or 4 different places while with 1.5 would only require to be updated in one place. But by no means does that mean that it is blocked, just that it requires more work as it is right now.

Reply:
Agreed. I guess my point that this PR adds a feature that doesn’t necessarily need to be added to other plugins _right now_ wasn’t clear. For example, neither of the operator-sdk’s own plugins support component configs yet the Go plugin does, and that disparity was acceptable at the time.

Also, a PR here shouldn’t be blocked by downstream stuff.

Reply:
My comment here was we will spend a lot of effort which shows unnecessary if we agree to merge the open prs. Then, if it is not urgent(can wait until Tue) why spend all this effort at all? However, it is fine.

/lgtm 

/hold 
to let you get the chance to merge when you wish.

Reply:
If I'm not mistaken this should be merged after #2071, right? It seems to have the same changes also inside here.

By the way this also has my LGTM. Even if phase 1.5 was implemented, we would have to do this here, so that doesn't make a difference.